### DISCUSSION TITLE
not able to build image from source, Pynq to setup zynq ultrascale+ board

### DISCUSSION THREAD

**Discussion by JaouaherBelgacem** [Xilinx/finn#947] [Q&A]
I tried to install the ultra96 image from pynq.io for my zynq ultrascale+ board but it did not work. When I looked for pynq images for the customized boards, I tried to follow the instructions in the https://pynq.readthedocs.io/en/latest/pynq_sd_card.html but when I tried this command source <path-to-vitis>/Vitis/2022.1/settings64.sh by changing the version and the path the file settings.64.sh does not exist in the installation file of Vitis.

**Comment by fpjentzsch** [comment]
Hi, this does not sound like a FINN related issue. You can probably find better support in the PYNQ forum: https://discuss.pynq.io/
